Source/EasySynth/Public/CoordinateSystemConverter.h

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2022 YDrive Inc. All rights reserved.
```

Comment:
Isn't the plan to remove this file altogether?

Reply:
It is removed in the `coordinate_systems` branch, will merge it here when we are sure it works correctly